verify/trust/trust.go

```diff
@@ -62,9 +85,15 @@ func (n *SimpleHTTPSGetter) Get(url string) (map[string][]string, []byte, error)
 	return header, body, nil
 }
 
```

Comment:
Why create ignored values?

Reply:
Sorry, I can't follow. Which values are you referring to?